scripts/mk.js

```diff
@@ -16034,6 +16059,28 @@ function move(getId, triggered) {
 				oKart.speed = oKart.maxspeed*cappedRelSpeed(oKart);
 				playIfShould(oKart,"musics/events/boost.mp3");
 			}
+			else if ((touche_etoile(fNewPosX, fNewPosY) || (fSelectedClass > 1.5 && touche_etoile(fMidPosX, fMidPosY))) && !oKart.tourne) {
```

Comment:
Maybe we should add a check that the player isn't using a bullet bill
<img width="323" alt="image" src="https://github.com/user-attachments/assets/aeb2463a-cf37-4cbc-be3f-13323be25776">